features-json/document-query-command-supported.json

```diff
@@ -0,0 +1,227 @@
+{
+  "title":"Document.queryCommandSupported()",
+  "description":"Reports whether or not the specified editor query command is supported by the browser.",
+  "spec":"https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html#querycommandsupported()",
+  "status":"ed",
```

Comment:
So `ed` doesn't stand for Editor's Draft. What should be put for that value? Maybe worth mentioning that in the contrib guide.